commcare_connect/opportunity/management/commands/auto_approval_opportunities.py

```diff
@@ -0,0 +1,27 @@
+from django.core.management import BaseCommand
+
+from commcare_connect.opportunity.models import Opportunity, OpportunityAccess
+from commcare_connect.opportunity.visit_import import update_payment_accrued
+
+
+class Command(BaseCommand):
+    help = "Run auto-approval logic over opportunity"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--opp", type=int, required=True, help="ID of the opportunity to run auto-approval logic on"
+        )
+
+    def handle(self, *args, **options):
```

Comment:
I think you can use `opp` as a named argument instead of getting it from options since it was added above.